doc/manuals/admin/database_model.md

```diff
@@ -210,7 +212,9 @@ Fields:
     following information:
     -   **entities**: an array containing a list of
         entities (mandatory). The JSON for each entity contains **id**,
-        **type** and **isPattern**.
+        **type**, **isPattern** and "isTypePattern". Note that, due to legacy
```

Comment:
Should `and "isTypePattern"` be bold, like `**type**, **isPattern**`   ?

Reply:
Right. I'll change.

Reply:
Fixed in d4bc65f